schemas/event.yml

```diff
@@ -261,15 +271,16 @@
             to indicate that a file access was attempted, but was not successful.
         - name: success
           description: >
-            Indicates that this event describes a successful result.  A common example is
+            Indicates that this event describes a successful result. A common example is
             `event.category:file AND event.type:create AND event.outcome:success`
             to indicate that a file was successfully created.
         - name: unknown
           description: >
             Indicates that this event describes only an attempt for which the result
-            is unknown. For example, if the event contains information only about a
-            request in an entity transaction that usually results in a response,
-            populating `event.outcome:unknown` is appropriate.
+            is unknown from the perspective of the event producer. For example, if the
+            event contains information only about a request in an entity transaction that
+            usually results in a response, populating `event.outcome:unknown` in the request
+            event is appropriate.
```

Comment:
We should clarify the description for "unknown" a bit, in my opinion. Here's an attempt:

> For example, if the event contains information only about the request side of a transaction that results in a response, populating `event.outcome:unknown` in the request event is appropriate.

We could also reinforce the distinction between absent and unknown, here as well. We don't need to reiterate all of [this last paragraph](https://github.com/elastic/ecs/pull/759/files?file-filters%5B%5D=.yml#diff-fd197febfa7b490a2fa19f86a95f6d52R262-R264), but perhaps just:

> This field should be left absent when an outcome doesn't make logical sense for the event.

Reply:
Thanks, updated accordingly.